agents/mox_cpr_abonnement/mox_cpr_abonnement/__main__.py

```diff
@@ -0,0 +1,68 @@
+# -- coding: utf-8 --
+#
+# Copyright (c) 2019, Magenta ApS
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import argparse
+import datetime
+import logging
+import os
+import json
+
+from mox_cpr_abonnement.lora_interface import (
+    lora_get_all_cpr_numbers,
+)
+from mox_cpr_abonnement.cpr_interface import (
+    cpr_add_subscription,
+    cpr_remove_subscription,
+    cpr_get_all_subscribed,
+)
+
+from mox_cpr_abonnement.settings import (
+    MOX_LOG_LEVEL,
+)
+
+# set warning-level for all loggers
+[
+    logging.getLogger(i).setLevel(logging.WARNING)
+    for i in logging.root.manager.loggerDict
+]
+
+logging.basicConfig(level=MOX_LOG_LEVEL)
+logger = logging.getLogger("mox_cpr_abonnement")
+logger.setLevel(logging.DEBUG)
+
+
+def update_cpr_subscriptions():
+    "add or remove subscriptions according to mora data"
```

Comment:
Skal starte med stort bogstav. Kør koden gennem `pep257` for at følge konventionerne for docstrings.